src/topology/sets/opens.lean

```diff
@@ -112,6 +112,21 @@ instance : inhabited (opens α) := ⟨∅⟩
 @[simp] lemma union_eq (U V : opens α) : U ∪ V = U ⊔ V := rfl
 @[simp] lemma empty_eq : (∅ : opens α) = ⊥ := rfl
 
+/-- The term of `opens X` corresponding to the complement of a singleton. -/
+@[simps] def compl_singleton [t1_space α] (x : α) : opens α :=
+⟨{x}ᶜ, is_closed_singleton.is_open_compl⟩
```

Comment:
I think it would be better to define `topological_space.closeds.singleton` and then use `topological_space.closeds.compl`, but that probably means moving it to another file since `closeds.lean` imports `opens.lean`. Then maybe the best is to start by proving that singletons are exactly the atoms of `closeds` and use that `compl` is an order-reversing bijection to get your statement about coatoms of `opens`. Does that sound right?

Reply:
I mean, I realize it feels a little backwards to talk about coatoms instead of atoms, but I don't think it's that important. If someone decides they want the result for closed sets then they can add it via the result on this PR and the order iso you just mentioned. I would say, let's not add it until we need it.

Reply:
Okay I'm fine about keeping the results only about coatoms of `opens` then. But I still think you should replace this particular definition by a `closeds.singleton` and then use `closeds.compl` on that, if that's not too annoying, because I think here the closed case will definitely be useful in itself.

Reply:
I have implemented what you suggested.